deployments/dev/images/secondary/install.R

```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/env r
+#!/usr/bin/env R
```

Comment:
I was wondering about this since it is technically correct, but hasn't been a problem. This script is evidently passed to R rather than made executable and run directly. The [upstream example](https://github.com/binder-examples/r/blob/HEAD/install.R) doesn't even have a hashbang.